genie-common-internal/src/main/java/com/netflix/genie/common/internal/dto/JobDirectoryManifest.java

```diff
@@ -319,6 +315,24 @@ private ManifestEntry buildEntry(
                 children
             );
         }
+
+        private String getMimeType(final String name, final Path path) {
+            // TODO: Loose conventions in here mostly for backwards compatibility but if there's empty
```

Comment:
We could/should move this `filename`->`mime type` map into configuration..

Reply:
That's a good idea! I'll add a TODO so can remember to come back to it